frontend/src/lib/utils.test.ts

```diff
@@ -101,3 +100,30 @@ describe("getCookie", () => {
     expect(cookie).toEqual("chocolatechip")
   })
 })
+
+describe("setCookie", () => {
+  afterEach(() => {
+    document.cookie.split(";").forEach(cookie => {
+      const eqPos = cookie.indexOf("=")
+      const name = eqPos > -1 ? cookie.substr(0, eqPos) : cookie
+      document.cookie = name + "=;expires=Thu, 01 Jan 1970 00:00:00 GMT"
+    })
```

Comment:
I'm not sure I understand this code. Can you add a comment explaining why it's needed? (Also, it looks like each iteration through this loop completely rewrites `document.cookie`, which possibly means that only the last iteration is needed?)